CODE/logicytics/Flag.py

```
@@ -579,15 +580,15 @@ def __exclusivity_logic(args: argparse.Namespace) -> bool:
         }
 
         if any(special_flags) and not any(action_flags):
-            print("Invalid combination of flags_list: Special and Action flag exclusivity issue.")
+            log.error("Invalid combination of flags_list: Special and Action flag exclusivity issue.")
```

Pylint (3.3.7)

[convention] Line too long (102/100)

(C0301)